versioned_docs/version-1.0.0/install-kubeslice/kubeslice-cli/command-reference.mdx

```diff
@@ -593,7 +593,7 @@ Use this command to install the required workloads to run KubeSlice Controller a
 ```
 kubeslice-cli --config <path-to-the-custom-topology-file> <install|i>
 kubeslice-cli install <command-options>
-kubeslice-cli install --profile=<minimal-demo|full-demo>
+kubeslice-cli install --profile=<minimal-demo|full-demo|enterprise-demo >
```

Comment:
Same question as below on `enterprise-demo` .. does it hold good for the OSS edition?